backend/tests/StorageContextTest.ts

```diff
@@ -0,0 +1,69 @@
+import { StorageContext } from '../storage/StorageContext'
```

Comment:
convention thing: when we write unit tests, we usually have the tests in the same directory as normal files, only with a different (".test.ts") suffix. so I you have /backend/blah/StorageContext.ts, the tests reside in /backend/blah/StorageContext.test.ts
I honestly like that better, since you usually edit tests along with the tested file and they are quite tighly coupled.

Reply:
I am more used to java like style where tests end up in a separate folder. This way it actually reminds me more C++ style! :) ok, I guess I don't mind either approach... and where do we store some test utils modules in this case? I tend recently to put some common util functions and data for tests into common modules and then import them in each test.

Reply:
maybe those should go to the aforementioned /test folder :)

Reply:
fixed in https://github.com/salsita/foosball-rating/pull/113/commits/d8bb46a15e9ce4e8733aba65b651d0ec0eece570